app/features/sendouq/routes/q.rules.tsx

```diff
@@ -110,13 +115,25 @@ export default function SendouqRules() {
 				another player&apos;s/team&apos;s ranking.
 			</div>
 
-			<h2 className="text-lg mt-4">Unsportsmanlike conduct</h2>
+			<h2 className="text-lg mt-4">Unsportsmanlike Conduct</h2>
 			<div>
 				It&apos;s not allowed to spawncamp the enemy without pushing the
 				objective provided it can&apos;t be considered a viable strategy taking
 				in account the game situation.
 			</div>
 
+			<h2 className="text-lg mt-4">Cheating / Mods</h2>
+			<div>
+				Cheating with any sort of modded hardware and or software is {" "}
+				<b>STRICTLY PROHIBITED</b>.
+			</div>
+
+			<h2 className="text-lg mt-4">Aliases</h2>
+			<div>
+				Aliases are allowed, Teams can ask for players to use their IGN.
+				If asked, all players must use the listed IGN on the user&apos;s profile.
+			</div>
```

Comment:
I don't really see the point of this. Seems redundant as you can use the FC to check you played against the correct player already.